tests/neo-vm.Tests/Tests/OpCodes/Control/CALLA.json

```diff
@@ -0,0 +1,148 @@
+{
+  "category": "Control",
+  "name": "CALLA",
+  "tests": [
+    {
+      "name": "Wrong type",
+      "script": "0x123A",
+      "steps": [
+        {
+          "actions": [
+            "StepInto"
+          ],
+          "result": {
+            "state": "Break",
+            "invocationStack": [
+              {
+                "instructionPointer": 1,
+                "nextInstruction": "CALLA",
+                "evaluationStack": [
+                  {
+                    "type": "Integer",
+                    "value": 2
+                  }
+                ]
+              }
+            ]
+          }
+        },
+        {
+          "actions": [
+            "StepInto"
+          ],
+          "result": {
+            "state": "FAULT"
+          }
+        }
+      ]
+    },
+    {
+      "name": "Real test",
+      "script": "0x0A070000003A661066",
```

Comment:
For general case, 32bits seem a lot Erik, are we sure this is the best way to go? It's nearly impossible to have an actual code that uses so much addressing... PUSHA and PUSHA_L may be an interesting alternative perhaps.

Reply:
I'm sure this is the best way.

Reply:
Well done then :+1: